collections/sentinel-s2-l2a-mosaic-120/README.MD

```diff
@@ -1,11 +1,46 @@
 # Sentinel-2 L2A 120m Mosaic
 
+## Short description
+
+Sentinel-2 L2A 120m mosaic is a derived product, which contains best pixel values for 10-daily periods, modelled by removing the cloudy pixels and then performing interpolation among remaining values. As clouds can be missed and as there are some parts of the world which have lengthy cloudy periods, clouds might be remaining in some parts. The actual modelling script is available [here](https://sentinel-hub.github.io/custom-scripts/sentinel-2/interpolated_time_series/).
+
```

Comment:
Text change should be also added here ("As clouds can be missed in **the removal process** and as there are...").